test/gtest/ucs/test_memtype_cache.cc

```diff
@@ -197,7 +197,15 @@ class test_memtype_cache : public ucs::test_with_param<ucs_memory_type_t> {
         }
     }
 
-private:
```

Comment:
private -> protected?

Reply:
this is not needed, `m_memtype_cache` has to be private